setup.py

```diff
@@ -7,4 +7,5 @@
         "rich>=10",
         "importlib-metadata; python_version < '3.8'",
     ],
+    package_data={"rich-click": ["py.typed"]},
```

Comment:
Bit late now, but maybe we could put this into `setup.cfg` instead? Most stuff is in there, it's only the requirements that are in `setup.py` so that they can be picked up by GitHub:

https://github.com/ewels/rich-click/blob/6d444bf9dd95c5b93e68da9fac695d4914116dbc/setup.cfg#L39